tests/console/yast2_nfs4_client.pm

```diff
@@ -71,6 +71,7 @@ sub run {
     type_string '10.0.2.101';
     # Explore the available shares and select the only available one
     send_key 'alt-e';
+    if (check_screen("console-messages-over-tty",10)) { record_soft_failure 'bsc#1011815' }
```

Comment:
you save some characters and get more clean line with post-fix notation:
`record_soft_failure 'bsc#1011815' if check_screen("console-messages-over-tty",10)`

Reply:
Please add verification where detection is visible, otherwise, it is better to delete this line and consider not to schedule for specific arch.

Reply:
It is for TW, not a specific arch, we see the soft-failure detected here: http://waaa-amazing.suse.cz/tests/10358#step/yast2_nfs4_client/40
It does not prevent the test from running though, but it is indeed a bug, so I assume it should be highlighted, at least. Even though that bug report is completely ignored by devs...

Reply:
I don't see the matching needle with some percentage in your link: http://waaa-amazing.suse.cz/tests/10358#step/yast2_nfs4_client/39. If there is no detection it is better not to include it as we'll forget to remove it, did you try needle with those characters over tty `[  ` ?

Reply:
Wrong run it seems, or maybe it is not visible every time. Here, we can see it http://waaa-amazing.suse.cz/tests/10360#step/yast2_nfs4_client/39

Reply:
nice! so at least we have some detection, you could also add some message on the `record_soft_failure`, to know what is about without clicking the link.